packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx

```diff
@@ -85,7 +87,19 @@ export const SynapseChat: React.FunctionComponent<SynapseChatProps> = ({
       },
     },
     (status: AsynchronousJobStatus<AgentChatRequest, AgentChatResponse>) => {
-      setCurrentProgressMessage(status?.progressMessage)
+      setCurrentlyProcessingJobId(status.jobId)
```

Comment:
when an agent chat message job is processing, enable the polling query for the trace events

Reply:
Since you aren't checking if the jobState is PROCESSING, I think you could remove this state variable and just use `isPending` returned by useSendChatMessageToAgent

Reply:
The trace requires the job ID.  Is there another (preferable) way to access the job ID?
Unclear if I need to wait for the job to be PROCESSING.  I think with the current code, the call to ask for the trace events would fail, but I don't think the failure will have an impact on the UI (am I wrong?).

Reply:
Totally missed that you're pulling the jobId out of the status, so that makes sense.

I think it should handle the failure case just fine, but we'll see